.github/workflows/ghcr_runtime.yml

```diff
@@ -191,7 +191,7 @@ jobs:
           # Print the full name of the image
           echo "Loaded Docker image: $image_name"
 
-          TEST_RUNTIME=${{ matrix.runtime_type }} SANDBOX_USER_ID=$(id -u) SANDBOX_CONTAINER_IMAGE=$image_name TEST_IN_CI=true poetry run pytest --cov=agenthub --cov=openhands --cov-report=xml -s ./tests/unit/test_runtime.py
+          TEST_RUNTIME=${{ matrix.runtime_type }} SANDBOX_USER_ID=$(id -u) SANDBOX_CONTAINER_IMAGE=$image_name TEST_IN_CI=true poetry run pytest --cov=agenthub --cov=openhands --cov-report=xml -s ./tests/ghcr_test_runtime.py
```

Comment:
Can we actually rename to `tests/runtime` folder? I was actually thinking we can break one really long `test_runtime.py` file into multiple smaller files, like `test_bash_cmd.py`, `test_ipython.py`, `test_browsing.py`, etc?